docker-compose.yml

```diff
@@ -25,6 +25,7 @@ services:
       - AB2D_DB_DATABASE=ab2d
       - AB2D_DB_USER=ab2d
       - AB2D_DB_PASSWORD=ab2d
+      - AB2D_EFS_MOUNT=/tmp/ab2d_efs_mount
```

Comment:
Is there some other script responsible for creating the ab2d_efs_mount directory or does it get created automatically somehow?

Reply:
@adaykin Worker does I believe.